samples/bluetooth/central_bas/src/main.c

```diff
@@ -270,6 +280,43 @@ static void button_handler(u32_t button_state, u32_t has_changed)
 }
 
 
+static void auth_cancel(struct bt_conn *conn)
+{
+	char addr[BT_ADDR_LE_STR_LEN];
+
+	bt_addr_le_to_str(bt_conn_get_dst(conn), addr, sizeof(addr));
+
+	printk("Pairing cancelled: %s\n", addr);
+}
+
+
+static void auth_done(struct bt_conn *conn)
+{
+	printk("%s()\n", __func__);
+	bt_conn_auth_pairing_confirm(conn);
+}
+
+
+static void pairing_complete(struct bt_conn *conn, bool bonded)
+{
+	printk("Paired conn: %p, bonded: %d\n", conn, bonded);
```

Comment:
Elsewhere we use address, I would suggest to continue to do that, instead of printing the pointer of the conn object.
Mixing pointer and address does not give us a very good debug log.

This goes for the rest of the pairing_* functions, and the rest of the samples.

Reply:
I aligned with other callbacks.